src/Mouf/Security/UserService/UserService.php

```diff
@@ -159,7 +159,7 @@ public function login($login, $password) {
 	 */
 	public function loginWithoutPassword($login) {
 		// First, if we are logged, let's unlog the user.
-		if (!$this->byPassIsLogged && $this->isLogged()) {
+		if ((!$this->byPassIsLogged || !isset($this->byPassIsLogged)) && $this->isLogged()) {

```

Comment:
You should never perform a "isset" on a $this->xxx field...
However, the field should ALWAYS be declared in the class, and it is not...